frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -133,12 +133,12 @@ class VolumeKeybindings {
   static getKeyboardControls() {
     return {
       c: () => Store.dispatch(createCellAction()),
-      v: () => {
+      v: guardNotTDViewport(() => {
         Store.dispatch(copySegmentationLayerAction());
-      },
-      "shift + v": () => {
+      }),
+      "shift + v": guardNotTDViewport(() => {
         Store.dispatch(copySegmentationLayerAction(true));
-      },
+      }),
```

Comment:
Why is this needed? As far as I can see, the corresponding `copySegmentationLayer` saga has

```
  const activeViewport = yield* select(state => state.viewModeData.plane.activeViewport);
  if (activeViewport === "TDView") {
    // Cannot copy labels from 3D view
    return;
  }
```

in its first lines.
Also, I cannot provoke any error triggering this functionality in the 3D viewport on master.

Reply:
> Also, I cannot provoke any error triggering this functionality in the 3D viewport on master.

Thanks for noticing :D. Looks like a messed up the testing I did on my own 🙈